app/style/app.css

```diff
@@ -11,6 +11,10 @@ body.loading {
   background: url(../img/gears.svg) 50% 50%;
 }
 
+gaia-header {
+  --header-icon-color: var(--title-color);
```

Comment:
Yeah, I'm not sure why the icon didn't use title-color in the web component, since that is what the text looks like.